tiff-2-png-converter.py

```diff
@@ -121,229 +270,68 @@ def convert_to_png(self):
             sequence_number = re.search("^p[0-9]{1,5}", file_name)
             sequence_number_int = int(sequence_number.group(0)[1:])
             if "ch1" in name:
```

Comment:
assuming the channels from the name is a pretty strong assumption and not very generalizable. It could be an input... but again, I'm confused how all of this changes if we are not producing rgb